infra/src/main/java/com/redhat/service/bridge/infra/k8s/ResourceEvent.java

```diff
@@ -0,0 +1,40 @@
+package com.redhat.service.bridge.infra.k8s;
+
+public class ResourceEvent {
```

Comment:
Any reason not to just pass in the entire resource, particularly if we have the `BridgeCustomResource` interface with `getType` and `getName` methods on it?